### PR TITLE
PT-2022: Ability to add notes to cancers in the pedigree

### PR DESCRIPTION
**Pull request by mjshepherd**

### CODE REVIEW

**Comment by mjshepherd** [comment]
@itaiGershtansky @danielpgross nominated

**Comment by itaiGershtansky** [comment]
+1

**Comment by danielpgross** [comment]
+1

**Comment by mjshepherd** [comment]
@sdumitriu

File: components/pedigree/resources/src/main/resources/pedigree/editor.css [hunk 1]
```diff
@@ -842,9 +842,23 @@
...
+.cancer_age_select {
+  display: inline-block;
+  margin-right: 6px
+}
+.cancer_field .clickable:hover {
+  cursor: pointer;
+}
+.cancer_field label.clickable.disabled {
+  color: #CCC;
+  cursor: initial;
```
Comment (sdumitriu) [inline components/pedigree/resources/src/main/resources/pedigree/editor.css]: `initial` is [not supported by IE](http://caniuse.com/#feat=css-initial-value), use one of `auto` or `default`.

File: components/pedigree/resources/src/main/resources/pedigree/editor.css [hunk 2]
```diff
@@ -842,9 +842,23 @@
...
+}
+.cancer_field textarea[name="cancers"] {
+    resize: vertical;
```
Comment (sdumitriu) [inline components/pedigree/resources/src/main/resources/pedigree/editor.css]: Wrong indentation.

File: components/pedigree/resources/src/main/resources/pedigree/editor.css [hunk 3]
```diff
@@ -842,9 +842,23 @@
...
+  margin-right: 6px
+}
+.cancer_field .clickable:hover {
+  cursor: pointer;
+}
+.cancer_field label.clickable.disabled {
+  color: #CCC;
+  cursor: initial;
+}
+.cancer_field textarea[name="cancers"] {
```
Comment (sdumitriu) [inline components/pedigree/resources/src/main/resources/pedigree/editor.css]: Why not use the simpler `.cancer_field textarea` selector? Are there any other textareas inside `cancer_field`? Shouldn't they all be resizeable?

**Comment by sdumitriu** [comment]
Bug: the button for adding notes isn't shown as enabled, although it is active, when closing and reopening the node menu.

To reproduce:
- click on a node, set Breast cancer as Affected
- the note button is black
- close the node menu
- click again on the node, the note button is gray
- clicking on the note button does show the notes textbox

**Comment by sdumitriu** [comment]
Bug: the notes aren't saved unless the user clicks somewhere in the node menu, but outside the textarea.

To reproduce:

- click on a node
- set Breast cancer as Affected
- click the notes button
- enter some text
- click outside the menu, for example on another node
- click again on the initial node
- the Affected status is preserved, but the notes are hidden and re-showing them brings up an empty textarea

**Comment by mjshepherd** [comment]
Good catches, sorry about that.

**Pull request merged**